packages/react-native-reanimated/.lintstagedrc.js

```diff
@@ -1,14 +1,5 @@
-const REANIMATED_PATH = 'packages/react-native-reanimated';
+const commonConfig = require('../../.lintstagedrc-common.js');
+
 module.exports = {
-  '*.(js|ts|tsx)': [
-    `cd ${REANIMATED_PATH} && yarn eslint `,
-    `cd ${REANIMATED_PATH} && yarn eslint --quiet --ext '.js,.ts,.tsx' src/`,
-    `cd ${REANIMATED_PATH} && yarn prettier --write`,
-  ],
-  'plugin/**/*.{js,ts,tsx}': `cd ${REANIMATED_PATH} && yarn lint:plugin`,
-  '**/*.{h,cpp}': `cd ${REANIMATED_PATH} && yarn lint:cpp`,
-  'android/src/**/*.java': `cd ${REANIMATED_PATH} && yarn format:java`,
-  'android/src/**/*.{h,cpp}': `cd ${REANIMATED_PATH} && yarn format:android`,
-  'apple/**/*.{h,m,mm,cpp}': `cd ${REANIMATED_PATH} && yarn format:ios`,
```

Comment:
Do I understand correctly that formatting will no longer be ran on pre-commit?

Reply:
It will no longer be attempted, since with syntax it only worked in specific cases. If you think it's a good idea to enforce formatting on pre-commit, I can add it to the pre-commit script in a reliable fashion. (but preferably in another PR)

Reply:
I think it's fine to have it this way, since formatting is enforced on the CI.